### ISSUE TITLE
Fatal exception in ghosts 7.0.0.0: System.NullReferenceException

### ISSUE THREAD

**Issue by iknowjason** [cmu-sei/GHOSTS#229]
Getting this error:  ```Fatal exception in ghosts 7.0.0.0: System.NullReferenceException```
Never seen this before.  Is this because I'm running it on Windows Server 2022 instead of supported Win10 OS?

If so I'd love to request Windows Server support for GHOSTS client.  The reason:  I'm trying to build my [PurpleCloud](https://www.purplecloud.network/) security lab with AWS support and adding GHOSTS automation to orchestrate building an unlimited number of clients with powershell, bash, and terraform.  This would increase the popularity of GHOSTS npc framework in Red and Blue team community.  AWS users would need to have Windows Server 2019 or 2022 support to run Ghosts clients because Windows 10/11 is BYOL in AWS.  I couldn't manage support for AMI building Windows 10 across all AWS regions for the entire community as a feature of the tool, without licenses.  Windows Server is already available in the AWS marketplace and trivial to use.  I'm currently using Windows Server 2022 in the security lab to simulate end user workstations with users logged in with Domain User credentials and Domain Join - works just fine.  It's too difficult and a management nightmare to support Windows 10 in AWS for most casual users.  Is this possible?  

Full Details:
```
PS C:\tools\ghosts\ghosts-client-x64-v7.0.0> .\ghosts.exe
             ('-. .-.               .-')    .-') _     .-')
            ( OO )  /              ( OO ). (  OO) )   ( OO ).
  ,----.    ,--. ,--. .-'),-----. (_)---\_)/     '._ (_)---\_)
 '  .-./-') |  | |  |( OO'  .-.  '/    _ | |'--...__)/    _ |
 |  |_( O- )|   .|  |/   |  | |  |\  :` `. '--.  .--'\  :` `.
 |  | .--, \|       |\_) |  |\|  | '..`''.)   |  |    '..`''.)
(|  | '. (_/|  .-.  |  \ |  | |  |.-._)   \   |  |   .-._)   \
 |  '--'  | |  | |  |   `'  '-'  '\       /   |  |   \       /
  `------'  `--' `--'     `-----'  `-----'    `--'    `-----'


GHOSTS (ghosts:7.0.0.0 [7.0.166.0]) running in production mode
Installed path: C:\tools\ghosts\ghosts-client-x64-v7.0.0
Running as Username: RTCAdmin - WindowsIdentity: EC2AMAZ-VJ4RN5F\RTCAdmin
C:\tools\ghosts\ghosts-client-x64-v7.0.0\config\application.json == True
Fatal exception in ghosts 7.0.0.0: System.NullReferenceException: Object reference not set to an instance of an object.
   at Ghosts.Client.Infrastructure.ClientConfigurationResolver.get_Dictionary() in C:\Projects\ghosts\src\Ghosts.Client\Infrastructure\ClientConfigurationResolver.cs:line 9
   at Ghosts.Client.Infrastructure.DebugManager.Evaluate() in C:\Projects\ghosts\src\Ghosts.Client\Infrastructure\DebugManager.cs:line 23
   at Ghosts.Client.Program.Run(String[] args) in C:\Projects\ghosts\src\Ghosts.Client\Program.cs:line 139
   at Ghosts.Client.Program.Main(String[] args) in C:\Projects\ghosts\src\Ghosts.Client\Program.cs:line 91
```

# Images

![ghosts1](https://github.com/cmu-sei/GHOSTS/assets/4335973/c556268c-9161-42db-b5c4-31ac4412d60e)

**Comment by sei-dupdyke** [comment]
Hi - GHOSTS should work fine on server (or any other versions of Windows that I'm aware of). On install, did you unzip the entire zip file? If you ran the ghosts.exe from the command line, did you do it from the folder where it resides (cd there first)?

**Comment by iknowjason** [comment]
Thanks for the details and fast response @sei-dupdyke .  I've confirmed it's some kind of issue with the zip files or config files I was automating with and not an actual issue with Ghosts.  I manually downloaded the zip with a browser on Windows Server and ran the ghosts.exe from the directory and it worked.  It looks like the zip file size was the same and I was running ghosts.exe from the extracted directory.  I'll debug and figure it out.  My initial thought to open up the ticket was because your docs say Windows 7 and 10 are supported and there is no mention of Windows Server support.  So my first thought is this was the problem.  It might be nice to adds that to your docs.  I did confirm that it runs just fine on Windows Server 2022 Datacenter addition.  Thanks again for the great work, I love this tool.  I'm adding it to an open source AWS security lab and I think it will help promote Ghosts.
